css/properties/scroll-snap-coordinate.json

```diff
@@ -18,23 +18,13 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": "39"
+              "version_added": "39",
+              "version_removed": "68"
+            },
+            "firefox_android": {
+              "version_added": "46",
```

Comment:
I would imagine that this would match with Firefox Desktop, though this seems to be a problem that existed in the data already.  I think the confusion is that while the flag existed, it was already set to `true` by default.  I'm thinking that we should set Firefox Android's `version_added` to "39" as well.

(I'll only comment on this one, though this applies to all files for the old properties!)